.github/workflows/nightly.yml

```
@@ -80,6 +80,19 @@ jobs:
       # run all tests by default
       testOnlyChangedCrates: false
 
+  tests-with-starfish:
+    if: ${{ inputs.rustTestsOnly || !inputs.simTestsOnly }}
```

```suggestion
    if: inputs.rustTestsOnly || !inputs.simTestsOnly
```
This should also work FYI